### PR TITLE
Fixes issue that prevented truncated markup from changing when underlying text changed

### PR DESCRIPTION
**Pull request by nlfurniss**
Now when the text passed to the component changes, the markup will too. Added test for this behavior.

### CODE REVIEW

File: addon/components/truncate-multiline.js [hunk 1]
```diff
@@ -116,16 +116,43 @@ export default Ember.Component.extend(ResizeHandlerMixin, {
   _didTruncate: false,
 
   /**
+   * Resets the component when attributes of a component have changed
+   * @return {Void}
+   */
+  didUpdateAttrs() {
+    this._super(...arguments);
+    this._resetState();
```
Comment (nickiaconis) [inline addon/components/truncate-multiline.js]: Should we check if the attribute that was updated is `text` before forcing a rerender? This gets called in other cases where an attribute is changed externally (i.e. external truncation control, hiding/revealing buttons, etc.), and I don't think we should trigger a rerender unless it's necessary.

**Comment by nickiaconis** [comment]
Otherwise, this looks ready to be merged. :+1:

**Comment by nlfurniss** [comment]
Updated (plus typo fixes)

**Comment by nickiaconis** [comment]
Fantastic! Thanks for the fix. 😀

**Pull request merged**